doc/api/child_process.markdown

```diff
@@ -34,7 +34,7 @@ the event loop until the spawned process either exits of is terminated.
 
 For convenience, the `child_process` module provides a handful of synchronous
 and asynchronous alternatives to `child_process.spawn()` and
-`child_process.spawnSync()`, each of which are documented fully [below][].
+`child_process.spawnSync()`, each of which are documented fully [here][].
```

Comment:
This doesn't entirely make sense. This links to the asynchronous process creation section. `spawnSync()` and company aren't documented there.